src/client/containers/404Page/404Page.Style.css

```diff
@@ -8,28 +8,28 @@
   width: 100%;
 }
 
-.page_bg {
+.page_404 .page_bg {
   background-image: url(https://cdn.dribbble.com/users/285475/screenshots/2083086/dribbble_1.gif);
   height: 400px;
   background-position: center;
 }
 
-.page_bg h1 {
+.page_404 .page_bg h1 {
   font-size: 80px;
 }
 
-.page_bg h3 {
+.page_404 .page_bg h3 {
   font-size: 80px;
 }
 
-.link_404 {
+.page_404 .link_404 {
   color: #fff !important;
   padding: 10px 20px;
   background: #39ac31;
   margin: 20px 0;
   display: inline-block;
 }
 
-.page_deatails {
+.page_404 .page_deatails {
```

Comment:
Why are you changing the pattern used for class names?
Don't you think it would be better to stick with the same pattern? ` e.g. `name-span`

Reply:
There are same names have been used in other components.. thats why i kept all component className infront of it.. 
this 404Page i created only like `name-span` but all pages changed ..so it would be easy to understand for everyoone..

Reply:
Fixed